.github/workflows/test.yaml

```diff
@@ -37,11 +37,11 @@ jobs:
             dependencies: highest
             allowed_to_fail: true
             variant: normal
-          - php-version: '7.3'
+          - php-version: '7.4'
             dependencies: highest
             allowed_to_fail: false
             variant: 'symfony/symfony:"4.4.*"'
-          - php-version: '7.3'
+          - php-version: '7.4'
             dependencies: highest
             allowed_to_fail: false
             variant: 'sonata-project/admin-bundle:"3.*"'
```

Comment:
you should change this to admin-bundle:"dev-master"

Reply:
If I do this, it will fail.

This change need to wait for https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1073